### PR TITLE
[ Website ]: Generate code snippets within gulp

### PR DESCRIPTION
**Pull request by rogeruiz**
## Description

Generate code snippets and preview snippets within the build pipeline instead of in the `docs/` directory.

Before you hit Submit, make sure you’ve done whichever of these applies to you:

- [x] Follow the [18F Front End Coding Style Guide](https://pages.18f.gov/frontend/) and [Accessibility Guide](https://pages.18f.gov/accessibility/checklist/).
- [x] Run `npm test` and make sure the tests for the files you have changed have passed.
- [x] Run your code through [HTML_CodeSniffer](http://squizlabs.github.io/HTML_CodeSniffer/) and make sure it’s error free.
- [x] Title your pull request using this format: [Website] - [UI component]: Brief statement describing what this pull request solves.

### CODE REVIEW

File: docs/_components/accordions.md [hunk 1]
```diff
@@ -7,129 +7,8 @@ lead: Accordions are a list of headers that can be clicked to hide or reveal add
...
-        <p>
-        No person shall be held to answer for a capital, or otherwise infamous crime, unless on a presentment or indictment of a Grand Jury, except in cases arising in the land or naval forces, or in the Militia, when in actual service in time of War or public danger; nor shall any person be subject for the same offence to be twice put in jeopardy of life or limb; nor shall be compelled in any criminal case to be a witness against himself, nor be deprived of life, liberty, or property, without due process of law; nor shall private property be taken for public use, without just compensation.
-        </p>
-      </div>
-    </li>
-  </ul>
-</div>
-
+{% include code/preview.html component="preview-accordion" %}
+{% include code/accordion.html component="code-accordion" %}
```
Comment (rogeruiz) [inline docs/_components/accordions.md]: These files exist in the repository, but the file names represented by the `component=""` parameter are generated using gulp.

**Comment by rogeruiz** [comment]
Tied to the issue #1353 and merging into #1368 once it's reviewed.

**Comment by rogeruiz** [comment]
cc: @msecret @maya @juliaelman @yozlet

**Comment by msecret** [comment]
THis looks good to me

**Pull request merged**